### PR TITLE
test: Add tests for missing /boot and --bootloader=none

### PR DESCRIPTION
**Pull request by henrywang**
Add two new integration tests:
- test-35-install-no-boot-dir: Tests install to-filesystem when /boot directory is absent
- test-36-install-bootloader-none: Tests install with --bootloader=none option to skip bootloader management

### CODE REVIEW

File: tmt/tests/booted/tap.nu [hunk 1]
```diff
@@ -18,3 +18,62 @@ export def is_composefs [] {
     let st = bootc status --json | from json
     $st.status.booted.composefs? != null
 }
+
+# Get the target image for install tests based on the running OS
```
Comment (cgwalters) [inline tmt/tests/booted/tap.nu]: Why can't we just use the image from `bootc status`?

(Of course with unified storage enabled it becomes suddenly simple and easy to actually run as a container as well via podman...getting closer hopefully)

File: tmt/tests/booted/test-install-bootloader-none.nu [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+let target_image = (tap get_target_image)
+
+def main [] {
+    tap begin "install with --bootloader=none"
+
+    truncate -s 10G disk.img
+
+    setenforce 0
+
+    systemd-run -p MountFlags=slave -qdPG -- /bin/sh -c $"
```
Comment (cgwalters) [inline tmt/tests/booted/test-install-bootloader-none.nu]: We're getting a lot of copy-pasta of this at this point, seems worth lowering into a helper in tap.nu

Comment (henrywang) [reply tmt/tests/booted/test-install-bootloader-none.nu]: Yes, totally agree.
